chain/client/src/client_actor.rs

```diff
@@ -1695,6 +1699,12 @@ pub fn start_client(
     #[cfg(feature = "test_features")] adv: Arc<RwLock<AdversarialControls>>,
 ) -> (Addr<ClientActor>, ArbiterHandle) {
     let client_arbiter_handle = Arbiter::current();
+    let mut rng_seed = [4; 32];
+    let thread_rng_seed = rand::thread_rng().next_u64().to_le_bytes().to_vec();
+    for i in 0..8 {
+        rng_seed[i] = thread_rng_seed[i];
+    }
```

Comment:
This piece of code looks very suspicious to me. What do you plan to achieve here? Why is `rng_seed` initialized as `[4;32]`?